src/views/schemaBrowser.ts

```diff
@@ -123,18 +123,25 @@ export default class schemaBrowser {
         if (node && node.contextValue === `schema`) {
           const schema = node.schema.toUpperCase();
 
-          const config = instance.getConfig();
+          const config = getInstance().getConfig();
           const currentLibrary = config.currentLibrary.toUpperCase();
   
           if (schema && schema !== currentLibrary) {
             config.currentLibrary = schema;
-            await instance.setConfig(config);
+            await getInstance().setConfig(config);
           }
 
           vscode.window.showInformationMessage(`Current schema set to ${schema}.`);
         }
       })
     )
+
+    getInstance().onEvent(`connected`, () => {
+      fetchSystemInfo().then(() => {
```

Comment:
The call to `fetchSystemInfo` is not required here.

Reply:
Fixed